### ISSUE TITLE
BLOCKED - Incomplete question description - Missing constraints - 327. Count of Range Sum

### ISSUE THREAD

**Issue by manoharreddyporeddy** [LeetCode-Feedback/LeetCode-Feedback#667]
#### Your LeetCode username
pgmreddy

#### Category of the bug
- [x] Question
- [ ] Solution
- [ ] Language

#### Description of the bug
Incomplete question description.
Missing constraints

327. Count of Range Sum
https://leetcode.com/problems/count-of-range-sum/

#### Code you used for Submit/Run operation

```
```

#### Language used for code
JS

#### Expected behavior

Expected:

327. Count of Range Sum
https://leetcode.com/problems/count-of-range-sum/

Constraints must be added to question description to avoid Time Limit Exceeded
Please add below:
- How many values can be there in `nums`?
- What is possible range of the values in `nums`?

Actual:
Constraints are missing in the question description.

**Comment by manoharreddyporeddy** [comment]
I have a solution ready.
It is passing correctly on small input.
Unable to submit as it is failing on large input of a million elements.

Please add constraints.

I am BLOCKED.

**Comment by lc-christina** [comment]
Hi @manoharreddyporeddy 
Thank you for reaching out to us. Would you mind sharing the problem number that you encountered the issue?

**Comment by manoharreddyporeddy** [comment]
@lc-christina 
Oops. Many of my questions are blocked so raised many bugs

Here it is:
327. Count of Range Sum
https://leetcode.com/problems/count-of-range-sum/

**Comment by manoharreddyporeddy** [comment]
Same constraints issue for, please have a look:

https://github.com/LeetCode-Feedback/LeetCode-Feedback/issues/683

**Comment by lc-pam** [comment]
Hi @manoharreddyporeddy 
Thank you for reaching out to us. I've relayed this issue to our team to investigate.

**Comment by lc-pam** [comment]
Hi @manoharreddyporeddy 
Thank you for your time. We've used your feedback to update the problem. Your LeetCode account has received 100 LeetCoins as the reward for this feedback. Since missing constraints is a well-known issue to the team, we'll not reward for any missing constraints report in the future but you're still welcome to file reports on github. We appreciate your support!

**Comment by manoharreddyporeddy** [comment]
@lc-pam 

1. Why was this issue closed - without UN BLOCKING me?
2. Please reopen & assign to someone who can understand a developer's issue better.

**Comment by lc-pam** [comment]
@manoharreddyporeddy The team has updated the problem and added constraints so I closed the ticket. Have you had the chance to check out the problem? If you still have question about the added constraints, please kindly leave them in the comments.

**Comment by manoharreddyporeddy** [comment]
@lc-pam 

Confusion is because:
- May be you replied in email with different problem number (697. Degree of an Array) instead of this problem number (327. Count of Range Sum)
- Or, did I get 100 leetcoins on both questions?


	--------------

	From: Pam (LeetCode) <support@leetcodehq.zendesk.com>
	Sent: Friday, July 24, 2020 1:56 AM
	Subject: [LeetCode] Re: Unclear question title - 697. Degree of an Array
	 
	##- Please type your reply above this line -##
	Your request (37087) has been updated. To add additional comments, reply to this email.

		
	Pam (LeetCode)

	Jul 23, 2020, 1:26 PM PDT

	Hi pgmreddy,

	Thank you for your time. We've used your feedback to update the problem.
	Your LeetCode account has received 100 LeetCoins as the reward for this feedback.
	We appreciate your support!

	Best,
	Pam

	--------------

**Comment by lc-pam** [comment]
Hi @manoharreddyporeddy 
You did get reward for both questions. The problem in the email is (697. Degree of an Array) and this ticket is (327. Count of Range Sum).

**Comment by manoharreddyporeddy** [comment]
Oh :)
LC din't give me popup for 1st +100 to me. 2nd +100 I got the popup.
Ok that might be another bug - if it repeats - for another day :)

Excellent. 😃 👍

**Comment by manoharreddyporeddy** [comment]
Just a note:

I have asked for below two:

1. How many values can be there in nums?       ---------- this is added
2. What is possible range of the values in nums?  ------- this is not added - I hope I can submit and check this is really needed.

Thanks

**Comment by manoharreddyporeddy** [comment]
@lc-pam

I have tried in 1 more way, but the Solution does not pass.

I have tried below methods with JavaScript
- Segment Tree - FAIL with TLE
- Prefix Sum - FAIL with TLE
- Sparse Table methods - FAIL with TLE

I understand that this is Hard question so it might take a few more tries.

However, I would to request answer for below question too  (which was asked originally above, in my initial request)

**- What is possible range of the values in nums?**

The answer will look something like below:

**-1000 <= nums[i] <= 1000**

In other words, I am still BLOCKED, because 2nd question (in bold above) I asked, is not answered yet.

Please let me know if you would re-open this issue, or, I can raise a new issue.

Thanks

**Comment by lc-pam** [comment]
Hi @manoharreddyporeddy ,

I've relayed the question 'What is possible range of the values in nums?' to the team and reopened the ticket.

**Comment by lc-pam** [comment]
Hi @manoharreddyporeddy ,
Thanks for your time. I've got updates from the team, for this particular problem the problem management team decided not to add the 2nd constraint. I recommend you check out accepted solutions in the discuss section.
